hail/python/hail/methods/impex.py

```diff
@@ -1664,6 +1667,9 @@ def import_matrix_table(paths,
         instead.
     delimiter : :obj:`str`
         A single character string which separates values in the file.
+    comment : :obj:`str` or :obj:`list` of :obj:`str`
+        Skip lines beginning with the given string if the string is a single
```

Comment:
> or :obj:`list` of :obj:`str`

Is a valid comment a ("str", "str2")?

Reply:
yes, that's fine -- used for two distinct comment patterns. Clarified docs.